recipes/phylommand/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: phylommand
+  version: {{ version }}
+
+source:
+  url: https://zenodo.org/record/200397/files/phylommand_v1.0.zip
+  md5: 14ae8509ff838530dcb50e7fb3ca48c1
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - nlopt
+  run:
+    - nlopt
+
+about:
+  home: https://github.com/mr-y/phylommand
+  license: GPLv3
+  license_file: LICENSE
+  summary: Command-line tools to create, manipulate, and/or analyze phylogenetic trees or pairwise alignments. Does not include "rudisvg" svg viewer.
```

Comment:
Try to be more brief - templates look better with 1 line for `summary` and details in `description`.